scss/modules/_clearfix.scss

```diff
@@ -0,0 +1,22 @@
+%vf-clearfix {
+  display: block;
+
+  &:after {
+    clear: both;
+    content: '\0020';
+    display: block;
+    height: 0;
+    overflow: hidden;
+    visibility: hidden;
+  }
+}
+
+/// Hard and soft clearing classes
+/// @group Grid
```

Comment:
I think this should be grouped in Utils or removed the @group to add it to General